### ISSUE TITLE
kontena master use notexisting gives ruby exception

### ISSUE THREAD

**Issue by matti** [kontena/kontena#1037]
```
$ kontena master use notexisting
Kontena error: no implicit conversion of String into Integer
```

Not nice error

**Comment by nevalla** [comment]
I think https://github.com/kontena/kontena/pull/997 will fix this issue.

**Comment by kke** [comment]
edge:

```
$ kontena master use foofoo                                                                                                                           Could not resolve master by name 'foofoo'.
For a list of known masters please run: kontena master list
```

**Comment by matti** [comment]
could have used my https://github.com/matti/contena to check what's up before creating this issue:

```
$ KONTENA_VERSION=edge contena master use lel
Unable to find image 'kontena/cli:edge' locally
edge: Pulling from kontena/cli

117f30b7ae3d: Already exists 
91c3c04b6fb4: Pull complete 
658b2bf8d105: Pull complete 
Digest: sha256:3a8b5158f18ef57fd95cb47966a487ab4ab9d8307d5e2dced533003209e42e93
Status: Downloaded newer image for kontena/cli:edge
Could not resolve master by name 'lel'.
For a list of known masters please run: kontena master list
```

**Comment by matti** [comment]
Fixed in release Release 0.15.5 https://github.com/kontena/kontena/pull/1054 with https://github.com/kontena/kontena/pull/997
